tldw_Server_API/app/core/TTS/adapters/pocket_tts_cpp_adapter.py

```
@@ -0,0 +1,576 @@
+class PocketTTSCppAdapter(TTSAdapter):
+    """Adapter for the PocketTTS.cpp command-line runtime."""
+
+    PROVIDER_KEY = "pocket_tts_cpp"
+    SUPPORTED_FORMATS: set[AudioFormat] = {
+        AudioFormat.MP3,
+        AudioFormat.WAV,
+        AudioFormat.OPUS,
+        AudioFormat.FLAC,
+        AudioFormat.PCM,
+        AudioFormat.AAC,
+    }
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)